app/models/animal.rb

```diff
@@ -19,7 +19,8 @@ class Animal < ActiveRecord::Base
 
   scope :searchable, -> { where(searchable: true) }
 
-  VALID_INTERESTS = %w(keep sell trade loan).freeze
+  VALID_INTERESTS = %w(keep sell trade loan sell\ or\ trade\ in\ 1-2\ years looking\ for\ mate).freeze
```

Comment:
Haha. I would too. I was actually trying to see if it would work like that. I'll swap it for a normal array.